lib/iris/tests/results/unit/merge/ProtoCube/register__CubeSig/noise.txt

```diff
@@ -4,4 +4,4 @@ failed to merge into a single cube.
   cube.attributes keys differ: 'stuffed'
   cube.cell_methods differ
   cube.shape differs: (3,) != (2,)
-  cube data dtype differs: int64 != float64
\ No newline at end of file
+  cube data dtype differs: int64 != int8
```

Comment:
So this was what it originally was, but as part of the previous dask work it was changed to `float64` because the cube data array was a masked array. However the masked array doesn't contain any masked values so the dask array wouldn't contain any nans and therefore could be int.